### ISSUE TITLE
Example with camera

### ISSUE THREAD

**Issue by KingWu** [zezo357/pytorch_lite#5]
Hi @zezo357 , just build an example for camera demo.

https://github.com/KingWu/pytorch_lite/tree/support_camera

I tried your function. but, it fail. Need your help.

```
E/flutter (25688): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: RangeError (index): Index out of range: index should be less than 76800: 76800
E/flutter (25688): #0      Uint32List.[] (dart:typed_data-patch/typed_data_patch.dart:2560:7)
E/flutter (25688): #1      Image.getPixel (package:image/src/image.dart:422:37)
E/flutter (25688): #2      ImageUtils.imageToByteListUint8 (package:pytorch_lite_example/utils/image_utils.dart:19:27)
E/flutter (25688): #3      _CameraViewState.onLatestImageAvailable (package:pytorch_lite_example/ui/camera_view.dart:124:37)
E/flutter (25688): #4      CameraController.startImageStream.<anonymous closure> (package:camera/src/camera_controller.dart:442:20)
E/flutter (25688): #5      _RootZone.runUnaryGuarded (dart:async/zone.dart:1586:10)
E/flutter (25688): #6      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:341:11)
E/flutter (25688): #7      _DelayedData.perform (dart:async/stream_impl.dart:591:14)
E/flutter (25688): #8      _StreamImplEvents.handleNext (dart:async/stream_impl.dart:706:11)
E/flutter (25688): #9      _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:663:7)
E/flutter (25688): #10     _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
E/flutter (25688): #11     _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
```

**Comment by zezo357** [comment]
Thanks for the code I will test and try to find a solution

**Comment by zezo357** [comment]
Fixed in version 1.1.0

you will find the example updated

**Comment by KingWu** [comment]
Awesome. @zezo357 you are hero!
